cpp/test/README.md

```diff
@@ -0,0 +1,8 @@
+# RRTMGPXX testing
+To run tests, execute the following:
+
+    ./get_data.sh
+    ./cmakescript.sh
+    cd build && make && ctest
+
+You will need to set some environment variables before running cmakescript.sh. Examples can be found in macbook.sh.
```

Comment:
Should we add a `source macbook.sh` step to the workflow here?

Reply:
Nah, I added text describing the workflow for summit instead.